fonts.css

```diff
@@ -0,0 +1,13 @@
+  @font-face {
+    font-family: "Montserrat", sans-serif;
+    src: url(C:\Users\atolo\Documents\Fonts\Montserrat-Regular.ttf);
+    font-weight: normal;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: "Oswald", sans-serif;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/shiny-tribble/pullRequest?prid=5485358)